code/modules/power/apc.dm

```diff
@@ -116,6 +116,25 @@
 
 /obj/machinery/power/apc/highcap/fifteen_k
 	cell_type = /obj/item/stock_parts/cell/high/plus
+	
+/obj/machinery/power/apc/auto_name
+	auto_name = 1
+	
+/obj/machinery/power/apc/auto_name/directional/north
```

Comment:
Not a fan of these. Pixel offsets are set automatically, so all you're actually doing is making subtypes for each dir.

Reply:
true, but the dir's alone are very helpful to me, especially as I make a lot of smaller maps, and manually setting the pixel offset and dir is very time consuming.

Reply:
I dislike the idea of making subtypes as a database of minor var edits but this seems to be the general consensus of map editors that this way is much easier and lacking any viable alternatives this may as well be fine

Reply:
How about dropping the /directional part of the path then since it's already nested under auto_name

Reply:
done